presto-main/src/main/java/com/facebook/presto/sql/planner/iterative/rule/PruneOrderByInAggregation.java

```diff
@@ -65,7 +65,13 @@ else if (functionRegistry.getAggregateFunctionImplementation(aggregation.getSign
             }
             else {
                 anyRewritten = true;
-                aggregations.put(entry.getKey(), new Aggregation(aggregation.getCall(), aggregation.getSignature(), aggregation.getMask(), ImmutableList.of(), ImmutableList.of()));
```

Comment:
formatting as separate commit?

Reply:
This is not a formatting change. I added the rewritten FunctionCall and removed unnecessary arguments from `new Aggregation(...)`